### PR TITLE
Windows support

### LINKED ISSUE #46: Test riff on Windows
Travis CI now has Windows available as a runtime. It would be nice to test the Windows CLI binary.

cc @fbiville

### PR DESCRIPTION
**Pull request by scothis**
Adds support for running riff on a Windows machine via GKE.

Known issues:
- kail is not compiled for windows and does not build from source
- eventing tests are disabled because they required kail to monitor logs
- local builds do not work, the running Docker daemon is only for Windows containers
- hyperv is not installed, virtualization extensions are disabled

Fixes #46

### CODE REVIEW

**Comment by scothis** [comment]
@trisberg what can Domino still do that FATS doesn't?

**Comment by trisberg** [comment]
Domino can

- install riff
- initialize namespace
- create Java and command functions with cloud builds
- create Node function with local build
- create simple eventing flow with hello + square functions and invoke it
- uninstall riff
- is easy to run from local terminal with existing cluster

I think FATS can do all of the above except for the last point

**Comment by scothis** [comment]
> I think FATS can do all of the above except for the last point

FATS itself doesn't install or uninstall riff, but provides configuration to those calls (like --node-port for minikube). There's no reason why FATS can't be used against an existing cluster, however, all of the current consumers create a cluster, install riff, run some functions, uninstall riff, cleanup the cluster. For CI, starting with a known clean cluster is beneficial.

What FATS can't do today which Domino can is run local function builds. AZP doesn't provide a Docker Daemon with a linux subsystem and installing linux support requires Hyper-v or VT-x which are disabled. We would have to run our own agent in order to automate that functionality.

File: .util.sh [hunk 1]
```diff
@@ -9,6 +9,21 @@ source `dirname "${BASH_SOURCE[0]}"`/.travis.sh
 
 ANSI_BLUE="\033[34;1m"
 
+unameOut="$(uname -s)"
+case "${unameOut}" in
+    Linux*)     machine=Linux;;
+    Darwin*)    machine=Mac;;
+    CYGWIN*)    machine=Cygwin;;
```
[the remark this answers is not in the document]
Comment (scothis) [reply .util.sh]: Done, also removed Mac/Darwin

**Pull request merged**